src/list-data/conversion-types.ts

```diff
@@ -5,7 +5,7 @@ export const ConversionTypes: ConversionTypesI[] = [
   {
     desc: 'Limited company to an unlimited liability company',
     text: 'Limited company to an unlimited liability company',
-    entity_type_cd: EntityType.UL,
+    entity_type_cd: EntityType.BC,
```

Comment:
I see you're changing the meaning of entity_type_cd to mean the "from" company instead of the "to" company...

Does the NR that get created have the correct fields set? Please compare with a conversion NR in Dev or Test environment.

Reply:
This is another thing we discussed: we need to compare a NR created with your code with a NR created in Dev and make sure all NR fields look the same.

Reply:
^^ lmk result of your testing.

Reply:
This will be specifically tested in #17614. Resolving.